views/pages/admin/get-involved/volunteer.pug

```diff
@@ -0,0 +1,22 @@
+extends /views/templates/admin-base.pug
+
+append styles
+    link(href="/assets/css/pages/admin/get-involved/volunteer.css" rel="stylesheet")
+
+append scripts
+    script(src="/assets/js/editor.js")
+    script.
```

Comment:
We should put this script into its own file, rather than have it inline in the view file.

Reply:
nvm lol, I think we maybe could later but it'll take more work than is worth blocking this PR for